tools/tsconfig/ts-library.json

```diff
@@ -0,0 +1,14 @@
+{
+    "$schema": "https://json.schemastore.org/tsconfig",
+    "display": "TypeScript Library",
+    "extends": "./base.json",
+    "compilerOptions": {
+        "lib": [
+            "dom",
```

Comment:
We remove the `jsx` support, however, through the absence of:

```
    "jsx": "react-jsx",
```

If we remove `dom` we lose some things we're using. There may be Node alternative imports for them, but it seems worth doing as a separate step.